### PR TITLE
Publication Importer fix Tv2

### PR DESCRIPTION
**Pull request by spficklin**
Issue #438 

## Type(s) of Change(s)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] API-specific change (fix or addition to an API function)
- [ ] Updates documentation (inline or markdown files)
This publication does not import properly:  https://www.ncbi.nlm.nih.gov/pubmed/?term=29792315.  The problem is that it has embedded HTML in the title and abstract (e.g. ```<i>Juglans</i>```).  That threw off the XML importer.  This PR fixes the problem.   This is the Tripal v2 fix that was applied in PR #459 for Tv3.

## Description

Use this drush command to import the publication:
```
drush trp-import-pubs --dbxref=PMID:29792315 --username=[username]
```
(substitute the [username] tag)

Tripal v2 should have automatically synced the publication So, go find the new publication page.  You should see it has a title and an abstract.  Previously the import would fail completely and not import or it would import partially and the title would be truncated and the abstract missing.

This is such a simple PR and it has already been tested by @bradfordcondon with issue #438 that I think this only needs on reviewer. 

## Additional Notes (if any):

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: tripal_feature/includes/tripal_feature.gff_loader.inc [hunk 1]
```diff
@@ -650,7 +650,12 @@ function tripal_feature_load_gff3($gff_file, $organism_id, $analysis_id,
         $strand = -1;
       }
       if (strcmp($phase, '.') == 0) {
-        $phase = '';
+        if ($type == 'CDS') {
```
Comment (bradfordcondon) [inline tripal_feature/includes/tripal_feature.gff_loader.inc:653]: This looks like code from another issue.  If you remove it I will approve and merge

Comment (spficklin) [reply tripal_feature/includes/tripal_feature.gff_loader.inc:653]: Oh there was no other issue. It was a fix that I copied from the Tv3 code that was missing in Tv2.   I didn't mean for that to get into this PR, but given that's it's the same as Tv3 code can we leave it in. so it gets merged.  Here's the same code in Tv3:

https://github.com/tripal/tripal/blob/44ba26f4bd17b5f52f6c042d0e9e58543db190fd/tripal_chado/includes/TripalImporter/GFF3Importer.inc#L565-L573

Comment (bradfordcondon) [reply tripal_feature/includes/tripal_feature.gff_loader.inc:653]: oh shoot im sorry, looks good then, lets merge.

Comment (spficklin) [reply tripal_feature/includes/tripal_feature.gff_loader.inc:653]: Oh, good call.  I'm glad you saw it.

**Comment by spficklin** [comment]
Thanks @bradfordcondon

**Pull request merged**